doc/rst/platforms/macosx.rst

```diff
@@ -4,40 +4,57 @@
 Using Chapel on Mac OS X
 ========================
 
-By and large, Chapel can be used on a Macintosh running OS X just like
-any other platform -- refer to the :ref:`chapelhome-quickstart` for more
-information on building and using Chapel.
+There are two main approaches for using Chapel on Mac OS X:
+
+1) Install via Homebrew_.  For Homebrew users, this is the
+   quickest way to get up and running, but it results in a copy of
+   Chapel that only supports shared-memory (single-locale) executions.
+
+2) Install and build Chapel from source, as with any other UN*X
+   system.  This is slightly more involved, but supports Chapel's full
+   feature set.  Refer to the :ref:`chapelhome-quickstart` for more
+   information.
 
 --------
 Homebrew
 --------
 
 A minimal installation of Chapel can be obtained through Homebrew_ with the
-following command::
+following commands::
 
+    brew update
     brew install chapel
 
-The above command installs a release version of Chapel. It is also
-possible to use Homebrew_ to build and install a development version
-based on the master branch on GitHub::
+These commands install the latest release of Chapel.  When using a
+Homebrew installation of Chapel, the ``CHPL_HOME`` directory is here::
 
-    brew install chapel --HEAD
+    /usr/local/Cellar/chapel/<chapel-version>/libexec/
 
-When using a Homebrew install of Chapel, the ``CHPL_HOME`` directory is
-here::
+Compile and run a test program::
 
-    /usr/local/Cellar/chapel/<chapel-version>/libexec/
+    chpl /usr/local/Cellar/chapel/<chapel-version>/libexec/hello.chpl
+    ./hello
+
+If you're new to Chapel, refer to the `What's Next?
+<https://chapel-lang.org/docs/usingchapel/QUICKSTART.html#what-s-next>`_
+section of :ref:`chapelhome-quickstart` for next steps.
+    
+.. note::
+
+   The Homebrew installation provides a minimal installation of Chapel
+   for users to explore and test the language.  Of the omitted
+   features, :ref:`multilocale <readme-multilocale>` support is most
+   notable.  Users interested in utilizing Chapel's complete set of
+   features should build Chapel from source (option 2 above).
 
 .. note::
 
-   The Homebrew installation provides a minimal installation of Chapel for
-   users to explore and test the language.
-   Of the omitted features, :ref:`multilocale <readme-multilocale>` support
-   is most notable.
-   Users interested in utilizing all of the
-   language's rich features should clone the repository_ or
-   download a release_, and :ref:`build Chapel from source <readme-building>`.
-
-.. _Homebrew: https://github.com/Homebrew/brew
-.. _repository: https://github.com/chapel-lang/chapel
-.. _release: https://github.com/chapel-lang/chapel/releases
+   It is also possible for developers to use Homebrew to build and
+   install a development version based on the master branch on
+   GitHub::
+
+    brew install chapel --HEAD
+
+
+.. _Homebrew: https://brew.sh/
+.. _whatsnext: 
```

Comment:
Is this "whatsnext" supposed to be here?  I don't see any references to it.

Reply:
Good point, it was a leftover (failed?) attempt that I ended up making a hyperlink within the body of the text itsef.  https://github.com/chapel-lang/chapel/pull/16475 removes it.